src/index.js

```diff
@@ -310,16 +311,22 @@ class DAPIClient {
   getUserByName(username) { return this.makeRequestToRandomDAPINode('getUser', { username }); }
 
   /**
-   * Sends serialized state transition header and data packet
-   * @param {string} rawStateTransition - hex string representing state transition header
-   * @param {string} rawSTPacket - hex string representing state transition data
-   * @returns {Promise<string>} - header id
+   * Send State Transition to machine
+   *
+   * @param {Transaction} stHeader
+   * @param {STPacket} stPacket
+   * @returns {Promise<!UpdateStateTransitionResponse>}
    */
-  sendRawTransition(rawStateTransition, rawSTPacket) {
-    return this.makeRequestToRandomDAPINode('sendRawTransition', {
-      rawSTPacket,
-      rawStateTransition,
-    });
+  async updateState(stHeader, stPacket) {
+    const stateTransition = new StateTransition();
+    stateTransition.setHeader(stHeader);
+    stateTransition.setPacket(stPacket);
+
+    const nodeToConnect = await this.MNDiscovery.getRandomMasternode();
+
+    const client = new CorePromiseClient(`${nodeToConnect.getIp()}:${this.getGrpcPort()}`);
```

Comment:
Hm, I guess we need to rename this client to something else. Too many cores in our system :slightly_smiling_face: But that's not a subject to this PR